docs/catalog.rst

```diff
@@ -1,5 +1,41 @@
 .. _sec_catalog:
 
+With ``stdpopsim``, you can run simulations from a number of bespoke demographic models
+that were chosen from notable manuscripts. These models have been rigorously checked 
+and tested by multiple people, so you can rest easy knowing that your simulations are
+reproducible and bug-free!
+
+This catalog shows you all of the possible bespoke simulations that
+you can configure and run with ``stdpopsim``.
+It is organised around a number of choices that you'll need to make:
+
+1. Which **chromosome**? (ie. which :class:`.Genome` object?)
```

Comment:
I considered this, but thought that including `Species` in the list here made it look as if all simulation options were compatible with all species. Latest version has a small edit to the text so that the `Species` objects are mentioned also.

Reply:
Looks good